contracts/bprotocol/Registry.sol

```diff
@@ -99,6 +99,8 @@ contract Registry is Ownable {
 
     function _newAvatar(address _owner) internal returns (address) {
         require(avatarOf[_owner] == address(0), "Registry: avatar-exits-for-owner");
+        // _owner should not be an avatar address
+        require(ownerOf[_owner] == address(0), "Registry: cannot-create-an-avatar-of-avatar");
```

Comment:
is this reachable? what does it mean to have an owner, and not to have an avatar? that you are an avatar yourself?